e2e-tests/e2eTestUtils/helpers.ts

```diff
@@ -21,6 +21,12 @@ export const ENV_MONITOR = {
   prod: "https://monitor.mozilla.org/",
 };
 
+export const FXA_HOSTS = {
+  stage: "accounts.stage.mozaws.net",
+  prod: "accounts.firefox.com",
+  dev: "accounts.dev.mozaws.net",
```

Comment:
Our dev server still uses the FXA stage server.

```suggestion
  dev: "accounts.stage.mozaws.net",
```